### PR TITLE
Remove multicast plugin as it's no longer supported in ES 5.

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: app/src/main/dist/config/crate.yml [hunk 1]
```diff
@@ -478,18 +477,11 @@
 #discovery.zen.publish_timeout: 30s
 
 # Unicast discovery allows to explicitly control which nodes will be used
-# to discover the cluster. It can be used when multicast is not present,
-# or to restrict the cluster communication-wise.
-# For example, Amazon Web Services doesn't support multicast discovery.
-# Therefore, you need to specify the instances you want to connect to a
-# cluster as described in the following steps:
+# to discover the cluster. Therefore, you need to specify the instances you want
+# to connect to a cluster as described in the following steps:
```
Comment (seut) [inline app/src/main/dist/config/crate.yml]: are these really still steps? Isn't this just one? if so, maybe rephrase the sentence..

File: app/src/main/dist/config/crate.yml [hunk 2]
```diff
@@ -478,18 +477,11 @@
...
-# 1. Disable multicast discovery (enabled by default):
-#
-#discovery.zen.ping.multicast.enabled: false
-#
-# 2. Configure an initial list of master nodes in the cluster
-#    to perform discovery when new nodes (master or data) are started:
+# Configure an initial list of master nodes in the cluster
+# to perform discovery when new nodes (master or data) are started:
 #
 # If you want to debug the discovery process, you can set a logger in
```
Comment (seut) [inline app/src/main/dist/config/crate.yml]: I think we should move this down below the list of nodes because there is a `:` at the end and then comes this sentence instead of the config section which is confusing... (could/should be done on master)

File: app/src/main/dist/config/crate.yml [hunk 3]
```diff
@@ -34,9 +34,8 @@
 #gateway.recover_after_nodes: 2
 #discovery.zen.minimum_master_nodes: 2
 
-# most cloud environments don't support multicast discovery
+# CrateDB doesn't support multicast discovery
```
Comment (mfussenegger) [inline app/src/main/dist/config/crate.yml]: I think the line should be removed completely. Without the context that it once did support multicast this doesn't make sense

File: app/src/main/dist/config/crate.yml [hunk 4]
```diff
@@ -34,9 +34,8 @@
 #gateway.recover_after_nodes: 2
 #discovery.zen.minimum_master_nodes: 2
 
-# most cloud environments don't support multicast discovery
+# Specify the instances that you want to form a cluster
```
Comment (dobe) [inline app/src/main/dist/config/crate.yml]: this is anyways a duplicate, see further down. i think it would make sense to look at the file as a whole and consider rephrasing some stuff, because if the multicast is removed, then actually unicast is THE discovery mechanism for non-cloud environments. so let's say the only standalone way to do discovery

File: blackbox/docs/installation.txt [hunk 5]
```diff
@@ -3,12 +3,8 @@ Installation
 ============
 
 This section describes how to install CrateDB on a single node. In order to
-get a CrateDB cluster up and running it is enough to repeat the process on
-another machine if the network supports multicast. The Nodes will discover and
-form a cluster automatically. Just like magic.
-
-In case the network doesn't support multicast, a unicast configuration is
-required. Please refer to our article on `Multi Node Setup`_ in order to learn
+get a CrateDB cluster up and running a unicast configuration is required.
```
Comment (dobe) [inline blackbox/docs/installation.txt]: a discovery configuration
